### ISSUE TITLE
joinWith relation

### ISSUE THREAD

**Issue by Velm14** [yiisoft/yii2#8474]
We can do joinWith such as (example):
```php
$tmp_news = News::find()
				->joinWith(['translate'=>function($q1){
					$q1->joinWith(['category']);
				}])->all();
```
but why we cannot do this : ( I mean first sql decreases count of queries, but second depth is not )
```php
$tmp_news = News::find()
				->joinWith(['translate'=>function($q1){
					$q1->joinWith(['category'=>function($q){
						$q->joinWith(['translate']);
					}]);
				}])->all();
```

**Comment by lynicidn** [comment]
try: `MyModel::find()->joinWidth(['user', 'user.profile', 'user.profile.contatcts'])`

**Comment by cebe** [comment]
can you rephrase the question? I do not get the problem...

**Comment by userlond** [comment]
Maybe there is typo: 
MyModel::find()->joinWith(['user', 'user.profile', 'user.profile.*contatcts*'])

**Comment by samdark** [comment]
No new info arrived for a long time. Closing.
